# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/palo/mkbootable.patch.cross
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- palo-2.24/Makefile.vanilla	2023-12-16 23:17:18.129562577 +0100
+++ palo-2.24/Makefile	2023-12-16 23:17:23.182562361 +0100
@@ -87,7 +87,7 @@
 		-s lifimage
 
 iplboot:	makeipl makepalo
-	./palo/mkbootable ipl/iplelf iplboot
+	./mkbootable ipl/iplelf iplboot
 
 clean:
 	# We intentionally don't remove iplboot here so that the file
